modules/virt-importing-vm-to-block-pv.adoc

```diff
@@ -78,4 +78,3 @@ spec:
 $ oc create -f <import-pv-datavolume.yaml><1>
```

Comment:
Fully resolving this might be out of scope, but the line between replaceable values and example values is also blurred here (and in `name: <import-pv-datavolume> <1>`)